docs/api-reference.rst

```diff
@@ -0,0 +1,4 @@
+API Reference
+=============
+
+[The following classes and methods constitute the Spanner DB API connection.]
```

Comment:
Why the brackets?

Reply:
This file was added merely as a sample "stub" and a reminder to write the actual docs. I've rephrased this a bit for now, but can also remove the file altogether.